.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,16 @@
-| Q             | A
-| ------------- | ---
-| Is bugfix?    | ✔️/❌
-| New feature?  | ✔️/❌
-| Breaks BC?    | ✔️/❌
+
+
+### What steps will reproduce the problem?
+
+### What is the expected result?
+
+### What do you get instead?
+
+
+### Additional info
+
+| Q                | A     |
+| ---------------- | ----- |
+| Version          | 1.0.? |
+| PHP version      |       |
+| Operating system |       |
```

Comment:
This is for issues. Doesn't make sense for pull requests.

Reply:
I reversed... 🤷🏻‍♂️🤦🏻‍♂️